include/mixer.h

```diff
@@ -34,31 +34,26 @@
 #include "audio_frame.h"
 #include "envelope.h"
 
-// Disable effc++ for Iir until its release.
+// Disable effc++ for IIR until its release.
 // Ref: https://github.com/berndporr/iir1/pull/39
 #if defined(__GNUC__)
-#	pragma GCC diagnostic push
-#	pragma GCC diagnostic ignored "-Weffc++"
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Weffc++"
 #endif
 #include <Iir.h>
 #if defined(__GNUC__)
-#	pragma GCC diagnostic pop
+#pragma GCC diagnostic pop
```

Comment:
Ahh, good news. That PR's been merged and there have been a couple IIR releases since then, so I think we should be OK to remove the `#if`s and `#pragma`s (if it passes CI, then :rocket: )

Reply:
Was wondering about that myself, yeah 😄